### PR TITLE
Report app.setUserTasks success/failure

### PR DESCRIPTION
**Pull request by miniak**
Changes return value of `Browser::SetUserTasks` from `void` to `bool`

### CODE REVIEW

File: atom/browser/browser.h [hunk 1]
```diff
@@ -156,7 +156,7 @@ class Browser : public WindowListObserver {
   };
 
   // Add a custom task to jump list.
-  void SetUserTasks(const std::vector<UserTask>& tasks);
+  bool SetUserTasks(const std::vector<UserTask>& tasks);
```
Comment (kevinsawicki) [inline atom/browser/browser.h]: Can you also update the docs to mention this method now returns whether it was successful? Thanks.

https://github.com/electron/electron/blob/master/docs/api/app.md#appsetusertaskstasks-windows

Comment (miniak) [inline atom/browser/browser.h]: done

Comment (miniak) [inline atom/browser/browser.h]: I have also noticed that both app.setAsDefaultProtocolClient(protocol) and app.removeAsDefaultProtocolClient(protocol) return a boolean, I've updated the docs.

**Comment by zcbenz** [comment]
:+1:

**Pull request merged**